### PR TITLE
OPGOPS-1808 - Jenkins2 image

### PR DESCRIPTION
**Pull request by carinadigital**
Containerised jenkins2 on our base image

### CODE REVIEW

**Comment by carinadigital** [comment]
Test run https://jenkins.service.opg.digital/job/OPG/job/docker-build-base/162/console

File: jenkins2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# Jenkins home directory is a volume, so configuration and build history
+# can be persisted and survive image upgrades
+VOLUME /var/jenkins_home
+
+# `/usr/share/jenkins/ref/` contains all reference configuration we want
+# to set on a fresh new installation. Use it to bundle additional plugins
+# or config file with your custom jenkins Docker image.
+RUN mkdir -p /usr/share/jenkins/ref/init.groovy.d
+
+ENV TINI_VERSION 0.14.0
```
Comment (tim-mcarthur-moj) [inline jenkins2/Dockerfile:28]: this should use the same service startup process as other images using /etc/sv and my_init

**Review by tim-mcarthur-moj — CHANGES_REQUESTED** [review]
There is a lot missing compared to the current jenkins 1 image.  This should just be an updated version of the original, with relevant changes for jenkins 2

File: jenkins2/Dockerfile (hunk 1 above)
Comment (carinadigital) [reply jenkins2/Dockerfile:28]: I was trying to maintain the upstream structure for easy upgrade. Looks like that's not going to fly.

**Pull request merged**